### PR TITLE
Adding GH actions doc

### PR DESCRIPTION
**Pull request by Oluane**
## Description

This PR adds a part about our Github Actions usage in the README.
 
## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?

X

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -392,7 +392,7 @@ const StyledComponent = styled.div {
 }
 ----
 
-Our chosen viewport breakpoints are : `576px` | `768px` | `992` | `1200`.
+Our chosen viewport breakpoints are : `576px` | `768px` | `992px` | `1200px`.
```
Comment (Fuzznimp) [inline README.adoc:395]: 🦅

File: README.adoc [hunk 2]
```diff
@@ -467,6 +467,22 @@ We are using https://github.com/testing-library/dom-testing-library[Testing Libr
...
 
+=== Github Actions
+
+To automate our test processes, we use https://docs.github.com/en/actions[Github Actions] to execute workflows, which allow us to run our test suites on new commit pushed to each pull request.
+
+Regarding our main flows for end to end tests (i.e `e2e-test-heroku.yml` & `lint-and-test.yml`), here is the gist of it. 
+Since we had to trigger different flows based on a third party event (i.e. `review app deployment`), we couldn't leverage Github checks management automation on PR.
+To do so, we first create manually a https://docs.github.com/en/rest/reference/checks#get-a-check-run[check-run] (`init-e2e-test-check` job inside `lint-and-test.yml`). This custom check-run is just initialized at this point, and its `status` is pending.
+Once we receive the status of the deployment from our third party actor, we run our end to end test suite (`e2e-tests-heroku` job inside `e2e-test-heroku.yml`) if the review app deployment is a success, or we skip it in the case of a failure during the process.
+The last job (`e2e-tests-check-run-status` job inside `e2e-test-heroku.yml`) is used to manage the status of the custom check-run we created previously.
```
Comment (Fuzznimp) [inline README.adoc:478]:
```suggestion
The last job (`e2e-tests-check-run-status` inside `e2e-test-heroku.yml`) is used to manage the status of the custom check-run we created previously. 
```

File: README.adoc [hunk 3]
```diff
@@ -467,6 +467,22 @@ We are using https://github.com/testing-library/dom-testing-library[Testing Libr
 
 We are using https://github.com/getgauge/taiko[Taiko] to test our workflows.
 
+=== Github Actions
+
+To automate our test processes, we use https://docs.github.com/en/actions[Github Actions] to execute workflows, which allow us to run our test suites on new commit pushed to each pull request.
+
+Regarding our main flows for end to end tests (i.e `e2e-test-heroku.yml` & `lint-and-test.yml`), here is the gist of it. 
+Since we had to trigger different flows based on a third party event (i.e. `review app deployment`), we couldn't leverage Github checks management automation on PR.
+To do so, we first create manually a https://docs.github.com/en/rest/reference/checks#get-a-check-run[check-run] (`init-e2e-test-check` job inside `lint-and-test.yml`). This custom check-run is just initialized at this point, and its `status` is pending.
```
Comment (Fuzznimp) [inline README.adoc:476]:
```suggestion
To do so, we first create manually a https://docs.github.com/en/rest/reference/checks#get-a-check-run[check-run] (`init-e2e-test-check` job inside `lint-and-test.yml`). This custom `check-run` is just initialized at this point, and its `status` is pending.
```

File: README.adoc [hunk 4]
```diff
@@ -467,6 +467,22 @@ We are using https://github.com/testing-library/dom-testing-library[Testing Libr
...
+Once we receive the status of the deployment from our third party actor, we run our end to end test suite (`e2e-tests-heroku` job inside `e2e-test-heroku.yml`) if the review app deployment is a success, or we skip it in the case of a failure during the process.
+The last job (`e2e-tests-check-run-status` job inside `e2e-test-heroku.yml`) is used to manage the status of the custom check-run we created previously. 
+If the deployment failed, we set its status to failure too. When the end to end test suite can be runned, we wait for it to complete before setting the custom check-run status according to the tests results.
```
Comment (Fuzznimp) [inline README.adoc:479]:
```suggestion
If the deployment failed, we set its status to failure too. When the end to end test suite can be runned, we wait for it to complete before setting the custom `check-run` status according to the tests results. 
```

**Review by Fuzznimp — COMMENTED** [review]
Nice reading, well written.

File: README.adoc (hunk 1 above)
Comment (Fenntasy) [reply README.adoc:395]: nice catch 👍

**Review by Fenntasy — APPROVED** [review]

**Pull request merged**